internal/organization/command_describe.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 type out struct {
-	IsCurrent bool   `human:"Current" serialized:"is_current"`
-	Id        string `human:"ID" serialized:"id"`
-	Name      string `human:"Name" serialized:"name"`
+	IsCurrent   bool   `human:"Current" serialized:"is_current"`
+	Id          string `human:"ID" serialized:"id"`
+	Name        string `human:"Name" serialized:"name"`
+	JIT_Enabled bool   `human:"JIT_Enabled" serialized:"jit_enabled"`
```

Comment:
```suggestion
	JitEnabled bool   `human:"JIT Enabled" serialized:"jit_enabled"`
```
We generally avoid underscores in our names. The human output shouldn't have underscores either.

Reply:
updated